src/components/common/PeopleAndRoles/OrgPerson.vue

```diff
@@ -219,11 +219,11 @@
                 <div class="mt-6">
                   <label class="sub-header">Mailing Address</label>
                   <div class="address-wrapper pt-4">
-                    <base-address
+                    <MailingAddress
                       ref="mailingAddressNew"
                       :editing="true"
-                      :schema="isPerson ? PersonAddressSchema :
-                        isChangeFiling ? OutOfBCAddressSchema : OfficeAddressSchema"
+                      :schema="isPerson ? DefaultAddressSchema :
+                        isChangeFiling ? InCanadaAddressSchema : InBcCanadaAddressSchema"
```

Comment:
This is a pretty good example of why I renamed the address schemas. (I actually clean this up further in my next PR.)